README.md

```diff
@@ -160,6 +162,14 @@ Results for applying pruning to reduce the parameters of the following models by
 | DenseNet121 [3] | 74.44                | 71.71                 |
 
 
+## Quantization Bag of Tricks
+
+If the quantization process degrades the model's accuarcy too much, check out the [Bag of Tricks](https://github.com/sony/model_optimization/tree/main/quantization_bag_of_tricks.md)
+for a few "tricks" for improving the quantization.
```

Comment:
Remove the "trick" terminology (everywhere).
Change every "bag of tricks" to "quantization troubleshooting" or "quantization accuracy troubleshooting" and here you can maybe change to "few advanced tools and things to look for, for improving the quantized model"